timhaneunsoo/MPC-Services
# Set Default Date to Current Date in Calendar

## Changes
### Features / Major Changes
- Added date provider functionality to ScheduleView
- Implemented proper date initialization in view's onAppear
- Added unit tests to verify date update behavior

### Testing Steps
1. Launch app before midnight
2. Verify calendar shows current date
3. Close app and reopen after midnight
4. Verify calendar updates to new date

## Related Issues
Fixes issue where the calendar would display the app's initial launch date instead of the current date
